package.json

```diff
@@ -53,6 +53,7 @@
     "acorn": "3.2.0",
     "ast-types": "0.8.16",
     "astring": "0.6.1",
+    "lodash": "4.13.1",
```

Comment:
Lodash likely will be a production dependency, moved it to `package.devDependies` for the time being anyway